blog/frontend/Week 7 Vue in depth.md

```diff
@@ -0,0 +1,33 @@
+# Vue in depth
+During this week we learned new advanced features of Vue to improve our skills in this framework and our code quality.
+
+## Vue in depth class
+### Advanced features of Vue and important knowledge
+
+*04-05-2022*
+
+<!-- (Do not change the line below!!!) -->
+| **Path Owner** | **Content Deliverer** | 
+| --- | --- | 
+| [Iván Palleiro](https://github.com/ivantxu) | [Luis Martínez](https://github.com/LuisMartinez15) | \ 
+
+\
+&nbsp; <!-- (Do not change this and above line PLEASE!!!) -->
+**Key learning points** <!-- (Do not change this line!!!) -->
+1. Slots
+2. Provide/Inject
+3. Watchers
+4. Binding
+5. Composable
+
+****
+
+## IMDb exercise
+(Autonomous work) <!-- Comment wheter if it is autonomous or group work -->
+The development of the frontend for the IMDb project in collaboration with the other Academy Path improved, having more information to display and implementing a Search Engine.
```

Comment:
Sorry for being a Pain in the ***, but could it be "Academy Paths" instead of "Academy Path"?

Reply:
Don't worry, it's already done ;)